shell/browser/api/electron_api_web_contents.cc

```diff
@@ -3136,11 +3137,15 @@ void WebContents::StartDrag(const gin_helper::Dictionary& item,
 
 v8::Local<v8::Promise> WebContents::CapturePage(gin::Arguments* args) {
   gfx::Rect rect;
+  bool stay_hidden = false;
+  bool stay_awake = false;
+
   gin_helper::Promise<gfx::Image> promise(args->isolate());
   v8::Local<v8::Promise> handle = promise.GetHandle();
 
-  // get rect arguments if they exist
   args->GetNext(&rect);
+  args->GetNext(&stay_hidden);
+  args->GetNext(&stay_awake);
```

Comment:
Having two booleans in a row is a minor wart, these arguments are easily swappable and users will always have to double-check to make sure they've placed them in the right order. (In the C++ world, clang-tidy has a [bugprone-easily-swappable-parameters](/clang/tools/extra/docs/clang-tidy/checks/bugprone-easily-swappable-parameters.html) check for this)

What if we take `args` to be either
- a single rect (for backwards compability)
- an options argument that takes a rect and the optional flags